src/integration/common/ads.canonical.js

```diff
@@ -18,7 +18,7 @@ export default () => {
       expect(adScript).toBeInTheDocument();
     });
 
-    it.only('dotcom config should be loaded', () => {
+    it('dotcom config should be loaded', () => {
```

Comment:
cc @holchris if we make this change on our branch (and remove the other describe.onlys), our homepage / radio schedule tests should run (and hopefully pass!!)